api/src/test/java/org/openmrs/CohortTest.java

```diff
@@ -165,4 +167,110 @@ public void setMemberIds_shouldSupportLargeCohorts() {
         double secondsToSet = (endTime - startTime)/1000;
         Assert.assertTrue("Setting cohort of size " + cohortSize + " took " + secondsToSet + " seconds", secondsToSet < 5);
     }
+    
+    @Test
+	public void hasActiveMembership_shouldFindAnActiveMemberCorrectly() throws Exception{
+	    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
Will this test pass after the year 2100?

Reply:
I will think of a way to create a Date that will always be in the future